Sources/Features/Breadcrumb/QueueFile.swift

```diff
@@ -1,5 +1,15 @@
 import Foundation
```

Comment:
Is it QueueFile? I believe its a generic queue instead. Can we rename the file to Queue or something like that?

Reply:
Queue it is!